src/components/button/button.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { storiesOf } from "@storybook/react"
+import { action } from "@storybook/addon-actions"
+import { text, boolean } from "@storybook/addon-knobs"
+import { readmeCleanup } from "../../../utils/readme"
+import { Button } from "."
+
+// @ts-ignore
+import readme from "./README.md"
+import { ButtonType } from "./button"
+import { Icon } from "../icon"
+
+const mockStory = storiesOf("Button", module)
+
+const subData = {
+  readme: {
+    sidebar: readmeCleanup(readme),
+  },
+  jest: ["button.test.tsx"],
+}
+
+mockStory.add(
+  "Button Default",
+  () => {
+    return (
+      <Button
+        disabled={boolean("Disabled", false)}
+        type={ButtonType.default}
```

Comment:
May be it will be ok, if we set `type` default prop?

Reply:
Yes by default the type is `ButtonType.default` so these line there can be omitted. 
I committed the omitted version so its made more clear.